changelog/added-manufacturer-code.md

```diff
@@ -0,0 +1 @@
+Added target manufacturer code to fix filtering on the website.
```

Comment:
This sounds way too general, and this change doesn't really need to be called out in the changelog anyway.

Reply:
Okay, removed!

Reply:
Thanks